lightly/models/modules/masked_vision_transformer_timm.py

```diff
@@ -20,14 +20,21 @@ class MaskedVisionTransformerTIMM(MaskedVisionTransformer, Module):
             The VisionTransformer object of TIMM.
         mask_token:
             The mask token.
+        weight_initialization:
+            The weight initialization method. Valid options are ['', 'skip']. 'skip'
+            skips the weight initialization and '' uses the default weight
```

Comment:
I would use the same order
```suggestion
            The weight initialization method. Valid options are ['skip', '']. 'skip'
            skips the weight initialization and '' uses the default weight.
```

Furthermore, why not use an enum?

Reply:
Updated :) I would like to keep the str instead of enum to be consistent with TIMM and the rest of the library. Enum makes it just a little bit harder to use because you have to remember to import it from the correct place etc. for very little gain.